crates/turbopack-tests/tests/snapshot/node/node_protocol_external/output/79fb1_turbopack-tests_tests_snapshot_node_node_protocol_external_input_index_8770aa.js

```diff
@@ -4,7 +4,7 @@
 "use strict";
 
 __turbopack_esm__({});
-var __TURBOPACK__external__node$3a$fs__ = __turbopack_external_require__("node:fs", true);
+var __TURBOPACK__url__external__node$3a$fs__ = __turbopack_external_require__("node:fs", true);
```

Comment:
this change seems kinda weird

Reply:
It's caused by the ident change. See my other comment